src/ioctls/mod.rs

```diff
@@ -25,7 +24,7 @@ pub mod vm;
 ///
 /// This typedef is generally used to avoid writing out io::Error directly and
 /// is otherwise a direct mapping to Result.
-pub type Result<T> = result::Result<T, io::Error>;
+//pub type Result<T> = result::Result<T, io::Error>;
```

Comment:
If we don't want to use this anymore, we should also delete the comment and the commented code.

Reply:
done